go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/fsnotify/fsnotify
```

Comment:
@nathany Was it an intentional change from https://gopkg.in/fsnotify.v1, or a mistake?

Reply:
An intentional change.